It introduced a new variable default_scheme to replace the old one
DEFAULT_AUTOPART_TYPE. Both of the default values are 'LVM'. But it is
overridden by 'BTRFS' in fedora.conf which is used as default profile for OE.
Because it fails to create home directory for new users, reset to use 'LVM' to
fix the issue and for backward compatile as well.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Update file path for anaconda 37.

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Rebase for anaconda 42 on 20250414.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 data/profile.d/fedora.conf | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/data/profile.d/fedora.conf b/data/profile.d/fedora.conf
index 6c074df73b..97112ad12d 100644
--- a/data/profile.d/fedora.conf
+++ b/data/profile.d/fedora.conf
@@ -14,10 +14,6 @@ default_on_boot = FIRST_WIRED_WITH_LINK
 [Bootloader]
 efi_dir = fedora
 
-[Storage]
-default_scheme = BTRFS
-btrfs_compression = zstd:1
-
 [User Interface]
 custom_stylesheet = /usr/share/anaconda/pixmaps/fedora.css
 
